src/mudlet-lua/lua/DB.lua

```diff
@@ -356,10 +358,12 @@ function db:_sql_values(values)
       s = "NULL"
     elseif t == "table" and v._timestamp ~= nil then
       if not v._timestamp then
-        return "NULL"
+        s = "NULL"
       else
         s = "datetime('" .. v._timestamp .. "', 'unixepoch')"
       end
+		elseif t == "table" and v._isNull then
```

Comment:
looks like the indentation got out of whack here somehow

Reply:
Was fiddling with that part on Notepad++, where I didn't have tabs to spaces on. Converted to spaces now.